CHANGELOG.md

```diff
@@ -51,6 +51,12 @@ Our versioning strategy is as follows:
 
 ### 🧹 Chores
 
+## 22.1.1
+
+### 🎉 New Features & Improvements
+
+* `[sitecore-jss]``[sitecore-jss-nextjs]` Pass `sc_layoutKind` to grapqhl layout request header. ([#1907](https://github.com/Sitecore/jss/pull/1907))
```

Comment:
I wouldn't say that it's a feature, since we are fixing the bug
```suggestion
* `[sitecore-jss]``[sitecore-jss-nextjs]` Pass `sc_layoutKind` to GraphQLEditingService request header to support shared/final editing layouts ([#1907](https://github.com/Sitecore/jss/pull/1907))
```